content/kubernetes/re-clusters/multi-namespace.md

```diff
@@ -0,0 +1,176 @@
+
+## Create role and role binding for managed namespaces
+
+Both the operator and the RedisEnterpriseCluster (REC) resource need access to each namespace the REC will manage. For each managed namespace, create a `role.yaml` and `role_binding.yaml` file within the managed namespace, as shown in the examples below.
+
+Replace `<rec-namespace>` with the namespace the REC resides in.
+Replace `<service-account-name>` with the your own value (by default this is the REC name).
+
+`role.yaml` example: 
+
+```yaml
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: redb-role
+  labels:
+    app: redis-enterprise
+rules:
+  - apiGroups:
+      - app.redislabs.com
+    resources: ["redisenterpriseclusters", "redisenterpriseclusters/status", "redisenterpriseclusters/finalizers",
+                "redisenterprisedatabases", "redisenterprisedatabases/status", "redisenterprisedatabases/finalizers"]
+    verbs: ["delete", "deletecollection", "get", "list", "patch", "create", "update", "watch"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["update", "get", "read", "list", "listallnamespaces", "watch", "watchlist",
+            "watchlistallnamespaces", "create","patch","replace","delete","deletecollection"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["get", "watch", "list", "update", "patch", "create", "delete"]
+```
+
+`role_binding.yaml` example:
+
+```yaml
+ kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: redb-role
+  labels:
+    app: redis-enterprise
+subjects:
+- kind: ServiceAccount
+  name: redis-enterprise-operator
+  namespace: <rec-namespace>
+- kind: ServiceAccount
+  name: redis-enterprise-admission
+  namespace: <rec-namespace>
+- kind: ServiceAccount
+  name: <service-account-name>
+  namespace: rec-namespace
+roleRef:
+  kind: Role
+  name: redb-role
+  apiGroup: rbac.authorization.k8s.io
+```
+
+Apply the files:
+
+```sh
+kubectl apply -f role.yaml
+kubectl apply -f role_binding.yaml
+```
+
+## Update Redis Enterprise operator ConfigMap
+
+There are two ways to update the operator ConfigMap (`operator-environment-config`) to specify which namespaces to manage.
+
+This ConfigMap can be created manually before deploying the RedisEnterpriseCluster, or will be created automatically when a Redis Enterprise cluster (REC) is deployed.
+
+{{<note>}}
+If the REC is configured to watch a namespace without setting the role and role binding permissions, or a namespace that's not created yet, the operator will fail and halt normal operations.
+{{</note>}}
+
+### Method 1: Namespace label (recommended)
+
+1. Create the `cluster_role_binding.yaml` and `cluster_role.yaml` files to add the namespace of the operator. Replace the `<rec-namespace>` with the namespace the Redis Enterprise cluster (REC) resides in.
```

Comment:
"to add the namespace of the operator" - not sure this part of the sentence is adding much. Please consider removing. Or, if you wish to explain what this is for, then "to grant the operator permissions to watch namespaces"